src/exo/API_scheduling.py

```diff
@@ -2170,6 +2502,12 @@ def eliminate_dead_code(proc, stmt_cursor):
 
     args:
         stmt_cursor       - cursor to the if or for statement
+        rc                - bool (whether to return relevant cursors)
+
+    relevant cursors:
+        ?block             - block within the if statement which is always executed
+        TODO: In trying to decide relevant cursors for this scheduling operation,
+        I think that this scheduling operation should really be split into two.
```

Comment:
I think you can omit any relevant cursors on the principle that this is very similar to a "delete statement"